x-pack/legacy/plugins/ingest/server/rest_api/init_api.ts

```diff
@@ -9,17 +9,33 @@ import { HapiFrameworkAdapter } from '../libs/adapters/framework/hapi_framework_
 import { ServerLibs } from '../libs/types';
 import {
   createGETPoliciesRoute,
-  createGETPoliciyRoute,
+  createGETPolicyRoute,
   createPOSTPoliciesRoute,
   createPUTPoliciesRoute,
+  createAddPolicyDatasourceRoute,
+  createRemovePolicyDatasourceRoute,
 } from './policy';
+import {
+  createGETDatasourcesRoute,
+  createGETDatasourceRoute,
+  createPOSTDatasourcesRoute,
+  createPUTDatasourcesRoute,
+} from './datasource';
 
 export function initRestApi(server: Server, libs: ServerLibs) {
   const frameworkAdapter = new HapiFrameworkAdapter(server);
 
-  // Policies route
-  frameworkAdapter.registerRoute(createGETPoliciyRoute(libs));
+  // Policies routes
+  frameworkAdapter.registerRoute(createGETPolicyRoute(libs));
   frameworkAdapter.registerRoute(createGETPoliciesRoute(libs));
   frameworkAdapter.registerRoute(createPOSTPoliciesRoute(libs));
   frameworkAdapter.registerRoute(createPUTPoliciesRoute(libs));
+  frameworkAdapter.registerRoute(createAddPolicyDatasourceRoute(libs));
+  frameworkAdapter.registerRoute(createRemovePolicyDatasourceRoute(libs));
+
+  // Datasources routes
```

Comment:
nitpick: I like to avoid comment and replace them by code when I can what about having `initRestApi` be something like 
```
function initRestApi() {
  registerPoliciesRoutes(frameworkAdapter, libs)
  registerDatasources(frameworkAdapter, libs)
```

Reply:
will change